### PR TITLE
[CS] solidity CS guidelines

### PR DESCRIPTION
**Pull request by n1c01a5**
fixing:
- indentation
- doxygen comments
- CS solidity
- use `view` instead of `constant`

### CODE REVIEW

**Comment by n1c01a5** [comment]
I follow this tuto https://truffleframework.com/tutorials/configuring-visual-studio-code for the solidity linting

File: contracts/Kleros.sol [hunk 1]
```diff
@@ -191,47 +200,48 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
 
-        lastPeriodChange=now;
+        lastPeriodChange = now;
         NewPeriod(period, session);
     }
 
 
-    /** @dev Activate tokens in order to have chances of being drawn. Note that once tokens are activated, there is no possibility of activating more.
+    /** @dev Activate tokens in order to have chances of being drawn. Note that once tokens are activated, 
+     *       there is no possibility of activating more.
```
Comment (clesaege) [inline contracts/Kleros.sol]: No indent in documentation style.

File: contracts/Kleros.sol [hunk 2]
```diff
@@ -247,150 +257,152 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
 
         msg.sender.transfer(_draws.length*dispute.arbitrationFeePerJuror);
     }
 
     /** @dev Execute all the token repartition.
-     *  Note that this function could consume to much gas if there is too much votes. It is O(v), where v is the number of votes for this dispute.
-     *  In the next version, there will also be a function to execute it in multiple calls (but note that one shot execution, if possible, is less expensive).
+     *       Note that this function could consume to much gas if there is too much votes. 
+     *       It is O(v), where v is the number of votes for this dispute.
+     *       In the next version, there will also be a function to execute it in multiple calls
```
Comment (clesaege) [inline contracts/Kleros.sol]: No indent in documentation style.

File: contracts/Kleros.sol [hunk 3]
```diff
@@ -247,150 +257,152 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
 
-    /* @dev Execute token repartition on a dispute for a specific number of votes.
-    *  This should only be called if oneShotTokenRepartition will throw because there are too many votes (will use too much gas).
-    *  NOTE There are 3 iterations per vote. e.g. A dispute with 1 appeal (2 sessions) and 3 votes per session will have 18 iterations
-    *  @param _disputeId ID of the dispute.
-    *  @param _maxIterations the maxium number of votes to repartition in this iteration
-    */
+    /** @dev Execute token repartition on a dispute for a specific number of votes.
+     *       This should only be called if oneShotTokenRepartition will throw because there are too many votes (will use too much gas).
+     *       Note There are 3 iterations per vote. e.g. A dispute with 1 appeal (2 sessions) and 3 votes per session will have 18 iterations
```
Comment (clesaege) [inline contracts/Kleros.sol]: No indent in documentation style.

File: contracts/Kleros.sol [hunk 4]
```diff
@@ -460,41 +472,41 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
     // **************************** //
 
     /** @dev Return the amount of jurors which are or will be drawn in the dispute.
-     *  The number of jurors is doubled and 1 is added at each appeal. We have proven the formula by recurrence.
-     *  This avoid having a variable number of jurors which would be updated in order to save gas.
+     *       The number of jurors is doubled and 1 is added at each appeal. We have proven the formula by recurrence.
+     *       This avoid having a variable number of jurors which would be updated in order to save gas.
```
Comment (clesaege) [inline contracts/Kleros.sol]: No indent in documentation style.

File: contracts/Kleros.sol [hunk 5]
```diff
@@ -460,41 +472,41 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
+    function amountJurors(uint _disputeID) public view returns (uint nbJurors) {
         Dispute storage dispute = disputes[_disputeID];
-        return (dispute.initialNumberJurors+1) * 2**dispute.appeals - 1;
+        return (dispute.initialNumberJurors+1) * 2 ** dispute.appeals - 1;
     }
 
     /** @dev Must be used to prove that a juror has been draw at least _draws.length times.
-     *  We have to require the user to specify the draws that lead the juror to be drawn.
-     *  Because doing otherwise (looping through all draws) could consume too much gas.
+     *       We have to require the user to specify the draws that lead the juror to be drawn.
```
Comment (clesaege) [inline contracts/Kleros.sol]: No indent in documentation style.

File: contracts/Kleros.sol [hunk 6]
```diff
@@ -553,54 +565,56 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
      *  @return fee Amount to be paid.
      */
-    function arbitrationCost(bytes _extraData) public constant returns(uint fee) {
+    function arbitrationCost(bytes _extraData) public view returns (uint fee) {
         return extraDataToNbJurors(_extraData) * arbitrationFeePerJuror;
     }
 
-    /** @dev Compute the cost of appeal. It is recommended not to increase it often, as it can be highly time and gas consuming for the arbitrated contracts to cope with fee augmentation.
+    /** @dev Compute the cost of appeal. It is recommended not to increase it often, 
+     *       as it can be highly time and gas consuming for the arbitrated contracts to cope with fee augmentation.
```
Comment (clesaege) [inline contracts/Kleros.sol]: No indent in documentation style.

**Comment by clesaege** [comment]
Solidity documentation does not use extra indents in doxygen comments (see p28 https://media.readthedocs.org/pdf/solidity/develop/solidity.pdf for example).
Other than that I'm OK.

**Comment by n1c01a5** [comment]
@clesaege doxygen comments fixing

File: contracts/Kleros.sol [hunk 7]
```diff
@@ -281,116 +291,123 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
-    *  @param _maxIterations the maxium number of votes to repartition in this iteration
-    */
+    /** @dev Execute token repartition on a dispute for a specific number of votes.
+<<<<<<< HEAD
+     *  This should only be called if oneShotTokenRepartition will throw because there are too many votes (will use too much gas).
+     *  NOTE There are 3 iterations per vote. e.g. A dispute with 1 appeal (2 sessions) and 3 votes per session will have 18 iterations
+=======
+     *       This should only be called if oneShotTokenRepartition will throw because there are too many votes (will use too much gas).
+     *       Note There are 3 iterations per vote. e.g. A dispute with 1 appeal (2 sessions) and 3 votes per session will have 18 iterations
+>>>>>>> 2dc3d9259a77fa6f3460956f6d028eb6ae177cfa
```
Comment (clesaege) [inline contracts/Kleros.sol]: What are those?

File: contracts/Kleros.sol [hunk 8]
```diff
@@ -281,116 +291,123 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
     }
 
-    /* @dev Execute token repartition on a dispute for a specific number of votes.
-    *  This should only be called if oneShotTokenRepartition will throw because there are too many votes (will use too much gas).
-    *  NOTE There are 3 iterations per vote. e.g. A dispute with 1 appeal (2 sessions) and 3 votes per session will have 18 iterations
-    *  @param _disputeId ID of the dispute.
-    *  @param _maxIterations the maxium number of votes to repartition in this iteration
-    */
+    /** @dev Execute token repartition on a dispute for a specific number of votes.
+<<<<<<< HEAD
```
Comment (clesaege) [inline contracts/Kleros.sol]: There is an issue there.

File: contracts/Kleros.sol [hunk 9]
```diff
@@ -281,116 +291,123 @@ contract Kleros is Arbitrator, ApproveAndCallFallBack {
...
+     *  This should only be called if oneShotTokenRepartition will throw because there are too many votes (will use too much gas).
+     *  NOTE There are 3 iterations per vote. e.g. A dispute with 1 appeal (2 sessions) and 3 votes per session will have 18 iterations
+=======
```
Comment (clesaege) [inline contracts/Kleros.sol]: And a last one there.

**Comment by n1c01a5** [comment]
see https://github.com/kleros/kleros/pull/65

**Pull request closed without merging**